static/app/utils/usePrismTokens.tsx

```diff
@@ -23,15 +23,18 @@ type IntermediateToken = {
   types: Set<string>;
 };
 
-const useLoadPrismLanguage = (language: string, {onLoad}: {onLoad: () => void}) => {
+const useLoadPrismLanguage = (
+  language: string,
+  {code, onLoad}: {code: string; onLoad: () => void}
+) => {
   const organization = useOrganization({allowNull: true});
 
   useEffect(() => {
-    if (!language) {
+    if (!language || !code) {
```

Comment:
Also added this check to return early if no code is actually there to be highlighted. That way we stop collected missing aliases for stack traces without source context.